scipy/linalg/basic.py

```diff
@@ -15,10 +15,11 @@
 from .decomp import _asarray_validated
 from . import decomp, decomp_svd
 from ._solve_toeplitz import levinson
+from ..fft import fft, ifft
```

Comment:
Making `linalg` depend on `fft` is new, and doesn't seem desirable. Better to have this import inside `matmul_toeplitz` perhaps?

Reply:
Moved the import into `matmul_toeplitz`. This adds around 1 microsecond of overhead per function call on my computer.